src/pages/contact.tsx

```diff
@@ -52,8 +52,8 @@ const ContactPage: React.FC<ContactPageProps> = (props: ContactPageProps) => {
               <SubTitle>E-Mail</SubTitle>
               <Trans i18nKey="contact.info-link">
                 <Text style={{ marginBottom: '40px' }}>
-                  Nutzen Sie unser Koaktformular oder schreiben Sie uns eine
-                  E-Mail an
+                  Nutzen Sie unser Kontaktformular oder schreiben Sie uns eine
+                  E-Mail an&nbsp;
```

Comment:
Did you ask yourself why the added `&nbsp;` is apparently not necessary for English? I'd rather not add this here. Let's figure out why it works in English and try to apply that same solution to German.